open_spiel/bots/stockfish/stockfish_bot.cc

```diff
@@ -0,0 +1,137 @@
+// Copyright 2019 DeepMind Technologies Ltd. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "open_spiel/bots/stockfish/stockfish/src/bitboard.h"
+#include "open_spiel/bots/stockfish/stockfish/src/endgame.h"
+#include "open_spiel/bots/stockfish/stockfish/src/position.h"
+#include "open_spiel/bots/stockfish/stockfish/src/search.h"
+#include "open_spiel/bots/stockfish/stockfish/src/thread.h"
+#include "open_spiel/bots/stockfish/stockfish/src/tt.h"
+#include "open_spiel/bots/stockfish/stockfish/src/uci.h"
+#include "open_spiel/bots/stockfish/stockfish/src/syzygy/tbprobe.h"
+#include "open_spiel/bots/stockfish/stockfish/src/types.h"
+#include "open_spiel/bots/stockfish/stockfish_bot.h"
+#include "open_spiel/spiel_utils.h"
+
+#include "open_spiel/games/chess.h"
+
+namespace open_spiel {
+namespace stockfish {
+namespace {
+
+// FEN string of the initial position, normal chess
+const char* StartFEN = "rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1";
+
+
+}  // namespace
+
+StockfishBot::StockfishBot(int argc, char **argv, int move_time) :
```

Comment:
Can you make a constructor which passes Options? I think that is what is parsed with CommandLine::init